app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = current_user.comments.new(
+      text: allowed_params[:text],
+      author_id: current_user.id,
+      post_id: @post.id
+    )
+    if @comment.save
+      redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", notice: 'Success!'
+    else
+      render :new, alert: 'Error occurred!'
+    end
+  end
+
```

Comment:
✅ A comment is created on behalf of the currently logged in user 👍🏻 

This can also be refactored to 
```ruby
def create
    @comment = @current_user.comments.new(params)
    @comment.post_id = params[:post_id]

    if @comment.save
      redirect_to post_path(@comment.post)
    else
      @article = Post.find(params[:post_id])
      redirect_to post_path(@post)
    end
  end
```